tests/pthread/test_pthread_run_script.cpp

```diff
@@ -1,19 +1,16 @@
+#include <assert.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <assert.h>
 #include <string.h>
 #include <emscripten/emscripten.h>
 #include <emscripten/threading.h>
 
-extern "C"
-{
-void EMSCRIPTEN_KEEPALIVE FinishTest(int result)
+extern "C" void EMSCRIPTEN_KEEPALIVE FinishTest(int result)
 {
   printf("Test finished, result: %d\n", result);
-#ifdef REPORT_RESULT
-  REPORT_RESULT(result);
-#endif
-}
+  assert(result == 1);
```

Comment:
Here too, is the assertion failure the expected way of failing?

Reply:
Yes we can use assert to fail best_exit tests test these days.   (I think that is true of btest too but I can't remember for sure).